README.md

```diff
@@ -3,7 +3,7 @@
 # Project Overview
 
 ## Secrets File
-An example one can be found in our private [living doc](https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-8442)
+An example one can be found in our private [living doc](https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-8442). It can go inside the root directory of the project. 
```

Comment:
It can, or it must?

Reply:
Must, I've changed it.

Reply:
It also must be called 'secrets.h` right? If so, could you please include that too?